mu4e/mu4e.texi

```diff
@@ -25,7 +25,7 @@ Documentation License.''
 @end copying
 
 @titlepage
-@title @t{Mu4e} - an e-mail client for GNU/Emacs
+@title @t{Mu4e} -- an e-mail client for GNU/Emacs
```

Comment:
If we stick to US English, I believe you're correct, an em-dash—without surrounding whitespaces—would be more appropriate.  However, I used en-dashes for consistency with the rest of the document where en-dashes seem to be more prevalent.

If there's consensus for “—”, I will change everywhere.